### ISSUE TITLE
Right-clicking some video files does not show option to add to playlist. 

### ISSUE THREAD

**Issue by InstantRedeemer** [clsid2/mpc-hc#861]
When I right click a .mp4, .mkv or .mov file I get the option to "add to MPC-HC playlist". 

But when I do the same for .wmv or .avi files I no longer get this option (I used to in the past).

**Comment by clsid2** [comment]
Explorer only shows that menu entry when MPC-HC is set as the default player for that file extension.

Step 1: Create file associations
Step 2: Set as default player in Windows settings

**Comment by InstantRedeemer** [comment]
I should of course have stated this in my original request, but obviously the correct associations are set in windows. And like I said this has been working correctly for all the years I have been using MPC-HC (and MPC before that). And if I double click the file with left mouse button MPC-HC opens and plays the file. I have also confirmed the same problem on 4 other PCs. 

![2021-01-24_11-30-37](https://user-images.githubusercontent.com/77888351/105627523-9c189100-5e37-11eb-868b-fa9dd53e426d.jpg)

**Comment by InstantRedeemer** [comment]
Hmm, I think I just found the cause (and solution) to this. I noticed the icons for those file types had the older (slightly smaller) MPC-HC icons. Even though I changed the associations to another program and then back to MPC-HC x64 they would come back as the older/other type (if I am not mistaken those were the icons I used when I had the last official build of MPC-HC). Even though I uninstalled that one when I installed this resurrected branch, these icons would still show up for these few file associations. I checked the registry high and low for a cause to this but couldn't find anything wrong, but I must have missed something.

When I manually clicked an .avi file and choose "open with" and then selected MPC-HC (again) the icon switched to the new type and the "add to MPC-HC playlist" option was finally back.
